README.md

```diff
@@ -44,6 +44,20 @@ Read our [versioning scheme](VERSIONING.md) for how we number and label releases
 We value those who take the time and effort to report security vulnerabilities.
 If you believe you have found a security vulnerability, please report it as outlined in our [Security and vulnerability disclosure policy](SECURITY.md).
 
+## :book: Docs
+
+The docs, including for older releases, are available [here](https://alan-turing-institute.github.io/data-safe-haven).
+
+To build the docs locally, check out the repo, navigate to the `docs` folder and `make` them:
+
+```{bash}
+git clone https://github.com/alan-turing-institute/data-safe-haven.git
+cd data-safe-haven/docs
+make html
+```
+
+This will add the contents to a folder called `_output` inside `docs`.
```

Comment:
It's worth pointing to the version of hosted on github.io but I'm not sure whether people who are viewing the front page of our repo will be interested in instructions about how to build a local copy of the docs.

Reply:
They might do if they want to PR a doc fix maybe? I could even say: "If you notice something looks wrong in the docs, fork, then clone, then modify and test with make, then PR" - but perhaps that should live in [contributing](https://github.com/alan-turing-institute/data-safe-haven/blob/develop/CONTRIBUTING.md)?

Reply:
Yes, probably somewhere in the contributing guide would be best.